packages/image_picker/image_picker_platform_interface/pubspec.yaml

```diff
@@ -3,20 +3,20 @@ description: A common platform interface for the image_picker plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/image_picker/image_picker_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.0.0-nullsafety
+version: 2.0.0
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.3.0-nullsafety.6
-  http: ^0.13.0-nullsafety.0
-  plugin_platform_interface: ^1.1.0-nullsafety.2
+  meta: ^1.3.0
+  http: ^0.13.0
+  plugin_platform_interface: ^2.0.0
```

Comment:
Please use ">=1.0.0 <3.0.0" so that we don't create unnecessary (in this case) segmentation between prerelease and stable NNBD plugins.

Reply:
done